test/lib/ufe/testUfePythonImport.py

```diff
@@ -26,13 +26,19 @@ class UfePythonImportTestCase(unittest.TestCase):
 
     def testImportModule(self):
         from mayaUsd import ufe as mayaUsdUfe
+        import ufe
 
         # Test calling a function that depends on USD. This exercises the
         # script module loader registry function and ensures that loading the
         # ufe library also loaded its dependencies (i.e. the core USD
         # libraries). We test the type name as a string to ensure that we're
         # not causing USD libraries to be loaded any other way.
-        invalidPrim = mayaUsdUfe.getPrimFromRawItem(0)
+        if ufe.VersionInfo.getMajorVersion() == 1:
```

Comment:
Sorry, I realized my previous code wasn't going to work once we switched to UFE v3 (with 0.3.0). UFE v1 always has major version 1.